code/game/machinery/doors/airlock_types.dm

```diff
@@ -649,8 +649,7 @@
 	return 0
 
 /obj/machinery/door/airlock/clockwork/brass
-	glass = TRUE
-	opacity = 0
+	glass = FALSE
```

Comment:
Default clock airlock isn't glass but for some reason these are used on the map instead of those

Reply:
The more you know